youtube_to_anki/anki.py

```diff
@@ -46,12 +58,16 @@ def make_package(
     """
     deck = Deck(deck_id=deck_id, name=deck_name)
     with TemporaryDirectory() as tempdir:
-        for i, (audio_chunk, transcript_chunk) in enumerate(
-            zip(audio_chunks, transcript_chunks)
+        for i, (audio_chunk, transcript_chunk, screenshot) in enumerate(
+            zip(audio_chunks, transcript_chunks, screenshots)
         ):
             audio_chunk.export(f"{tempdir}/{i}_{deck_id}.mp3")
-            note = _make_note(transcript_chunk["text"], f"{i}_{deck_id}.mp3")
+            if isinstance(screenshot, ndarray):
```

Comment:
somehow Python has an issue comparing `screenshot != -1` if it's an `ndarray`, it throws a runtime exception! I would expect it to just return `true`, since the type of the variables is different (that is how it works in most languages). The `isinstance` workaround seems to be easy enough & works.